pkg/csi/cinder/nodeserver.go

```diff
@@ -157,16 +253,35 @@ func (ns *nodeServer) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 	if len(volumeID) == 0 {
 		return nil, status.Error(codes.InvalidArgument, "NodeUnpublishVolume volumeID must be provided")
 	}
-	_, err := ns.Cloud.GetVolume(volumeID)
+
+	ephemeralVolume := false
```

Comment:
Does volume_context doesn't have the field ephemeral = true or not?

Reply:
during NodeUnpublishVolume, we do not get info from kubernetes weather it is for inline or normal volume, we need to determine it by ourselves.

Reply:
ok